README.md

```diff
@@ -136,11 +136,14 @@ For additional support, see our [org-wide support policy](https://github.com/Eas
 # Install dependencies
 make install
 
+# Install style guide (Unix only)
+make install-style
```

Comment:
thought: if `make install` installed the styleguide, there wouldn't be a need to document this one-off command. We'd just reuse it under the hood. Less surface area to worry about